src/packages/pvm/pvm/args-decoder.ts

```diff
@@ -0,0 +1,499 @@
+import { ArgumentType, Mask } from "@typeberry/pvm-debugger-adapter";
+import { ExtendedWitdthImmediateDecoder } from "@/packages/pvm/decoders/extended-with-immediate-decoder.ts";
+import { ImmediateDecoder } from "../decoders/immediate-decoder";
+import { NibblesDecoder } from "../decoders/nibbles-decoder";
```

Comment:
if you don't change anything in the decoders logic, you can use them from pvm-debugger-adapter package (I can export them if they aren't exported)